assignments/array-methods.js

```diff
@@ -56,28 +56,54 @@ const runners = [{"id":1,"first_name":"Charmain","last_name":"Seiler","email":"c
 // ==== Challenge 1: Use .forEach() ====
 // The event director needs both the first and last names of each runner for their running bibs.  Combine both the first and last names into a new array called fullName. 
 let fullName = [];
+runners.forEach((x) => {
+fullName.push(x.first_name + " " + x.last_name);
+});
 console.log(fullName);
 
 // ==== Challenge 2: Use .map() ====
 // The event director needs to have all the runner's first names converted to uppercase because the director BECAME DRUNK WITH POWER. Convert each first name into all caps and log the result
 let allCaps = [];
+const fNM = runners.map(x => x.first_name);
+joined = fNM.join()
+caps = joined.toUpperCase()
+allCaps = caps.split(",")
 console.log(allCaps); 
 
```

Comment:
Interesting approach. Gets the job done and thats what is important. My solution looked more like the one Josh did:
```
let allCaps = runners.map((person) => {
  return person.first_name.toUpperCase();
});
console.log(allCaps); 
```